src/main/java/Problem2/SortList.java

```diff
@@ -17,12 +17,35 @@ public static ListNode sortList(ListNode head) {
     }
 
     public static ListNode findMidAndBreak(ListNode head) {
-        // homework
+        // Runs in 1 pass
+        ListNode slow_ptr = head;
+        ListNode fast_ptr = head;
+        ListNode prev_slow_ptr = null;
+        if (head != null) {
```

Comment:
same here. reverse the condition check.

```java
if (head == null) {
    return null;
}
```

and the rest of the function do not need to be in a {} block.